### PR TITLE
User/improvements

### PR DESCRIPTION
**Pull request by williamollio**
**TODO's left**
- MUI warnings show up in console when friend request done
- Bug tab name when create dm chat
- Implement cache for status and pictures
- warning profile view uncontrolled inputs
- Endpoints Protection : Additional check on the BE
- Response from server : Return accordingly the error code and error message for each case

### CODE REVIEW

File: ui/src/views/Profile/ProfileView.tsx [hunk 1]
```diff
@@ -232,105 +152,24 @@ export default function ProfileView(props: Props): React.ReactElement {
...
-                            <CustomMultiSelect
-                              label={t(translationKeys.addFriends)}
-                              options={users}
-                              onChange={onChange}
-                              selectedValues={value}
-                            />
-                          );
-                        }}
+                    <Box sx={{ width: "40%" }}>
+                      <CustomTextField
```
Comment (hskowron) [inline ui/src/views/Profile/ProfileView.tsx:166]: The Textfield is clickable on readonly view, the name does not actually change or anything but I am wondering if we might want to just set the Textfield to disabled there or find some other way to make it not interactable.

Comment (williamollio) [reply ui/src/views/Profile/ProfileView.tsx:166]: @hskowron Yeah I have set the readonly prop on the textfield but I can find a way in my next to disable the click event

**Comment by hskowron** [comment]
> Bug tab name when create dm chat

What bug is that?

> warning profile view uncontrolled inputs

I think that is easy to fix, do you want me to push a change for that?

**Comment by williamollio** [comment]
I need to run more tests because it happened one time for me when clicking on the icon on the left drawer for opening a dm chat

**Comment by williamollio** [comment]
If you know how to fix it, yes pls

**Review by hskowron — APPROVED** [review]
There are a couple of places where you didn't use the translations for some toast titles and Tooltips, you could add those at some point if you want. ;D

**Comment by williamollio** [comment]
Yeah will do for the message error, I don't think that it's required for the title

**Pull request merged**